docs/how-to-build-with-ddn/overview.mdx

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 1
+sidebar_label: Basics
+description: "Learn how to get started with Hasura DDN and your GraphQL API."
+keywords:
+  - hasura ddn
+  - graphql api
+  - quickstart
+  - getting started
+  - guide
+---
+
+import { OverviewTopSectionIconNoVideo } from "@site/src/components/OverviewTopSectionIconNoVideo";
+import { OverviewPlainCard } from "@site/src/components/OverviewPlainCard";
+import Icon from "@site/static/icons/speedometer-04.svg";
+import Link from "@docusaurus/Link";
+
+# Basics
+
+## Introduction
+
+Hasura is a powerful-yet-simple-to-create API. Each tutorial in this section will guide you through creating your first
+Hasura DDN API on top of your preferred data source.
+
+### You have data
+
+Data is a broad term. What's our definition? It can be anything. No, really: anything.
+
+We realize that data lives in a variety of places—relational databases, non-relational databases, vector stores, OLAP,
+OLTP, and even generated on-the-fly by your functions...and you probably want to access all of it from a single
+endpoint.
+
+### You connect it to Hasura
+
+Regardless of where your data lives, you'll connect it to your API using a **native data connector**, or data connector
+for short. These connectors will introspect your data sources and generate metadata that Hasura DDN will use to generate
```

Comment:
```suggestion
Regardless of where your data lives, you'll connect it to your API using a **native data connector**. These connectors will introspect your data sources and generate metadata that Hasura DDN will use to generate
```

This we can safely leave out the "or data connector for short" especially since immediately thereafter we are using the term "connector" not "data connector".